apps/ledger-live-desktop/src/renderer/components/RecoverRestore/index.tsx

```diff
@@ -53,7 +55,20 @@ const RecoverRestore = () => {
         }
       },
       error: (error: Error) => {
-        setError(error);
+        if (
+          (error instanceof TransportStatusError && error.message.includes("0x6d06")) ||
+          error instanceof DeviceNotOnboarded
```

Comment:
I would prefer if we did an util with the code from this so we also support stax error https://github.com/LedgerHQ/ledger-live/blob/8fe0797008950ba25d966442f2767a932ebf87c1/apps/ledger-live-desktop/src/renderer/components/DeviceAction/index.tsx#L437-L440